Build failure: mwprocapture

### Nixpkgs version

- Stable (24.11)

### Steps to reproduce

  boot.kernelPackages = pkgs.linuxPackages_latest;

  hardware.mwProCapture.enable = true;


### Can Hydra reproduce this build failure?

No, Hydra cannot reproduce this build failure.

### Link to Hydra build job

_No response_

### Relevant log output

```console
error: builder for '/nix/store/86505q5kwq8xm44n3mz06jpq5g6b5x1i-mwprocapture-1.3.0.4390-6.14.3.drv' failed with exit code 2;
       last 25 log lines:
       > cp ProCaptureLib_64.lib ProCaptureLib.o
       > touch .ProCaptureLib.o.cmd
       > make -C /nix/store/891qcsv4m0plkrqcpgk3p1qxbqbs97vy-linux-6.14.3-dev/lib/modules/6.14.3/build M=/build/ProCaptureForLinux_4390/src modules
       > make[1]: Entering directory '/nix/store/891qcsv4m0plkrqcpgk3p1qxbqbs97vy-linux-6.14.3-dev/lib/modules/6.14.3/build'
       > make[2]: Entering directory '/build/ProCaptureForLinux_4390/src'
       >   CC [M]  sources/dma/mw-dma-mem.o
       >   CC [M]  sources/dma/mw-dma-user.o
       >   CC [M]  sources/dma/mw-dma-kernel.o
       >   CC [M]  sources/dma/mw-dma-phy.o
       >   CC [M]  sources/ospi/ospi-linux.o
       >   CC [M]  sources/ospi/linux-file.o
       >   CC [M]  sources/avstream/v4l2.o
       >   CC [M]  sources/avstream/v4l2-sg-buf.o
       >   CC [M]  sources/avstream/mw-event-dev.o
       > sources/avstream/mw-event-dev.c:135:16: error: 'no_llseek' undeclared here (not in a function); did you mean 'noop_llseek'?
       >   135 |     .llseek  = no_llseek,
       >       |                ^~~~~~~~~
       >       |                noop_llseek
       > make[4]: *** [/nix/store/891qcsv4m0plkrqcpgk3p1qxbqbs97vy-linux-6.14.3-dev/lib/modules/6.14.3/source/scripts/Makefile.build:207: sources/avstream/mw-event-dev.o] Error 1
       > make[3]: *** [/nix/store/891qcsv4m0plkrqcpgk3p1qxbqbs97vy-linux-6.14.3-dev/lib/modules/6.14.3/source/Makefile:1997: .] Error 2
       > make[2]: *** [/nix/store/891qcsv4m0plkrqcpgk3p1qxbqbs97vy-linux-6.14.3-dev/lib/modules/6.14.3/source/Makefile:251: __sub-make] Error 2
       > make[2]: Leaving directory '/build/ProCaptureForLinux_4390/src'
       > make[1]: *** [/nix/store/891qcsv4m0plkrqcpgk3p1qxbqbs97vy-linux-6.14.3-dev/lib/modules/6.14.3/source/Makefile:251: __sub-make] Error 2
       > make[1]: Leaving directory '/nix/store/891qcsv4m0plkrqcpgk3p1qxbqbs97vy-linux-6.14.3-dev/lib/modules/6.14.3/build'
       > make: *** [Makefile:33: all] Error 2
       For full logs, run 'nix log /nix/store/86505q5kwq8xm44n3mz06jpq5g6b5x1i-mwprocapture-1.3.0.4390-6.14.3.drv'.
error: 1 dependencies of derivation '/nix/store/j3khqdmsmwmak3m86z46qswxli1fhqzk-etc-modprobe.d-nixos.conf.drv' failed to build
error: 1 dependencies of derivation '/nix/store/1l2awv5gvmmdcg9m74l2q60jbq242gam-linux-6.14.3-modules.drv' failed to build
error: 1 dependencies of derivation '/nix/store/vbd19wac22d338492z2gz2acym7a0hm8-man-paths.drv' failed to build
error: 1 dependencies of derivation '/nix/store/jsc512rdir3azd7gf172xwkha9779xgs-mwprocapture-1.3.0.4390-6.14.3_fish-completions.drv' failed to build
error: 1 dependencies of derivation '/nix/store/adq4ny50hsnhbjm9g74rrvalfg8hxczk-system-path.drv' failed to build
error: 1 dependencies of derivation '/nix/store/yrh0q16a6p8ckjavg5mdc2fcz2hc123i-nixos-system-niedzwiedz-main-24.11.20250424.5630cf1.drv' failed to build
```

### Additional context

I think this is the same procedure as @SuperSandro2000 did before, I can make a PR but I'm not sure how to test this

### System metadata

✦ ❯ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.87, NixOS, 24.11 (Vicuna), 24.11.20250424.5630cf1`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.14`
 - nixpkgs: `/nix/store/wxb0mwz1mgrg49d47rwnbdfldsgl2k83-source`

### Notify maintainers

[@MP2E
](https://github.com/MP2E)
---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/os-specific/linux/mwprocapture/default.nix
+++ b/pkgs/os-specific/linux/mwprocapture/default.nix
@@ -18,12 +18,12 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "mwprocapture";
-  subVersion = "4390";
-  version = "1.3.0.${subVersion}-${kernel.version}";
+  subVersion = "1.3.4418";
+  version = "${subVersion}-${kernel.version}";
 
   src = fetchurl {
     url = "https://www.magewell.com/files/drivers/ProCaptureForLinux_${subVersion}.tar.gz";
-    sha256 = "sha256-a2cU7PYQh1KR5eeMhMNx2Sc3HHd7QvCG9+BoJyVPp1Y=";
+    sha256 = "sha256-ZUqJkARhaMo9aZOtUMEdiHEbEq10lJO6MkGjEDnfx1g=";
   };
 
   nativeBuildInputs = kernel.moduleBuildDependencies;
@@ -65,11 +65,12 @@ stdenv.mkDerivation rec {
       "$out"/bin/mwcap-info
   '';
 
-  meta = with lib; {
+  meta = {
     homepage = "https://www.magewell.com/";
     description = "Linux driver for the Magewell Pro Capture family";
-    license = licenses.unfreeRedistributable;
-    maintainers = with maintainers; [ flexiondotorg ];
-    platforms = platforms.linux;
+    license = lib.licenses.unfreeRedistributable;
+    maintainers = with lib.maintainers; [ flexiondotorg ];
+    platforms = lib.platforms.linux;
+    broken = lib.versionAtLeast kernel.version "6.15";
   };
 }

